lms/templates/api/oauth2/client_side_redirect.html.jinja2

```diff
@@ -0,0 +1,11 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Redirecting to {{ link_text }}</title>
+    <meta http-equiv="refresh" content="0; url={{ redirect_url }}">
+  </head>
+  <body>
+  <p>Redirecting to <a href="{{ redirect_url }}">{{ link_text }}</a></p>.
+  </body>
+</html>
```

Comment:
See notes in `lms/views/api/canvas_studio.py` for why this exists. It needs better styling, but I will add that in future.